modules/installation-generate-aws-user-infra-ignition.adoc

```diff
@@ -61,6 +62,49 @@ Because you create and manage the worker machines yourself, you do not need
 to initialize these machines.
 endif::[]
 
+ifeval::["{context}" == "installing-gcp-user-infra"]
+. Remove the Kubernetes manifest files that define the worker machines:
++
+----
+rm -f openshift/99_openshift-cluster-api_worker-machineset-*.yaml
+----
++
+Because you create and manage the worker machines yourself, you do not need
+to initialize these machines.
+
+. Currently, emptying the compute pools makes control plane machines schedulable. However, due to a link:https://github.com/kubernetes/kubernetes/issues/65618[Kubernetes limitation], router Pods running on control plane machines will not be reachable by the ingress load balancer.
++
+Ensure the `mastersSchedulable` parameter is set to `false` in the
+`manifests/cluster-scheduler-02-config.yml` scheduler configuration file to keep
+router Pods and other workloads off the control plane machines.
+
+. Optional: If you do not want
+link:https://github.com/openshift/cluster-ingress-operator[the Ingress Operator]
+to create DNS records on your behalf, remove the `privateZone` and `publicZone`
+sections from the `manifests/cluster-dns-02-config.yml` DNS configuration file:
++
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: DNS
+metadata:
+  creationTimestamp: null
+  name: cluster
+spec:
+  baseDomain: example.openshift.com
+  privateZone: <1>
+    tags:
+      Name: mycluster-100419-mmhhx-int
+      kubernetes.io/cluster/mycluster-100419-mmhhx: owned
```

Comment:
Is this accurate for GCP?  I thought `kubernetes.io/cluster/{id}` was AWS-specific.

Reply:
Not accurate for GCP. It looks more like the following. Though, does that really matter for this specific step where we are removing it anyhow?
```
spec:
  baseDomain: example.openshift.com
  privateZone:
    id: mycluster-100419-private-zone
  publicZone:
    id: example.openshift.com
status: {}
```